jira-macro/jira-macro-default/src/main/java/org/xwiki/contrib/jira/macro/JIRADisplayer.java

```diff
@@ -43,7 +44,22 @@ public interface JIRADisplayer
      * @param parameters the macro parameters specified by the user; can be used to specify displayer-specific
      *            configuration data (for example displayers allows the user to specify the list of JIRA issue fields to
      *            display
-     * @return the list of Blocks rerpesenting what to display
+     * @return the list of Blocks representing what to display
      */
     List<Block> display(Collection<Element> issues, JIRAMacroParameters parameters);
+
+    /**
+     * Displays the passed JIRA issues based on the macro context.
+     * @param issues the issues to display
+     * @param parameters the macro parameters specified by the user; can be used to specify displayer-specific
+     *            configuration data (for example displayers allows the user to specify the list of JIRA issue fields to
+     *            display
+     * @param context the transformation context of the macro.
+     * @return a list of blocks representing what to display
+     */
+    default List<Block> display(Collection<Element> issues, JIRAMacroParameters parameters,
+        MacroTransformationContext context)
```

Comment:
Sounds good. I'd add a `@since 10.1` in the javadoc to indicate when it was added. We don't currentmy use `@Unstable` in extensions (AFAIR it would require some work) but we could work `@since`.